src/System.Globalization/ref/System.Globalization.cs

```diff
@@ -15,88 +15,6 @@ public static partial class CharUnicodeInfo
         public static System.Globalization.UnicodeCategory GetUnicodeCategory(char ch) { return default(System.Globalization.UnicodeCategory); }
```

Comment:
CharUnicodeInfo and UnicodeCategory can be moved down as they will be exposed by Char.GetUnicodeCategory methods